src/Template/Element/short_description.ctp

```diff
@@ -42,7 +42,7 @@
         </div>
     </md-toolbar>
 
-    <div class="container">
+    <div class="container" id="langing-page-search" ng-cloak>
```

Comment:
The id isn't used anywhere else but nevertheless I think there is a typo and it should be `landing-page-search`.